src/components/InfoTodo/InfoTodo.tsx

```diff
@@ -0,0 +1,89 @@
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { updateTodo, deleteTodo } from '../../api/todos';
+
+type Props = {
+  todo: Todo;
+  handleLoadTodos: () => void;
+  isEditing?: boolean;
+};
+
+export const InfoTodo: React.FC<Props> = ({
+  todo,
+  handleLoadTodos,
+  isEditing,
+}) => {
+  const { title, completed } = todo;
+  const [isLoading, setIsLoading] = useState(false);
+  const [value, setValue] = useState(title);
+  const [isDoubleClick, setIsDoubleClick] = useState(false);
+
+  return (
+    <div data-cy="Todo" className={classNames('todo', { completed })}>
+      <label className="todo__status-label">
+        <input
+          data-cy="TodoStatus"
+          type="checkbox"
+          className="todo__status"
+          onChange={async () => {
+            setIsLoading(true);
+            await updateTodo(todo.id, { completed: !completed });
+            setIsLoading(false);
+            handleLoadTodos();
+          }}
```

Comment:
Create handler
It's ok if you set one value to state on click, but in this case it's not ok, we shouldn't write all our logic inside jsx